docs/algorithms/bc.rst

```diff
@@ -1,8 +1,12 @@
+.. _behavioral cloning:
+
 =======================
 Behavioral Cloning (BC)
 =======================
 
-Supervised learning on observation-action pairs.
+Behavioral cloning directly learns a policy by using supervised learning on
+observation-action pairs from expert demonstrations. It is a simple approach to learning
+a policy, but the policy often generalizes poorly and does not recover well from errors.
```

Comment:
Consider linking to DAgger algorithm (like BC but gathers on-policy demonstrations), and GAIL/AIRL (more robust approaches to learning from demonstrations).